hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -227,16 +227,14 @@
 
   // List of test specifications.
   //
-  // If you are adding a test that has been generated from the `sw_tests`
-  // dictionary declared in `sw/device/tests/meson.build`, the `sw_images` list
-  // below should contain `sw/device/tests/<sw_test_name>` (without any more
-  // subdirectories) because that is where the meson target is created. For
-  // example `gpio_test` is added to `sw_tests` in
-  // `sw/device/tests/sim_dv/meson.build`, but the final meson targets all
-  // start with `sw/device/tests/gpio_test_`.
+  // If you are adding a test that has been generated from a Bazel
+  // `opentitan_functest` macro, you can specify the test using the path to the
+  // Bazel target. For example, if the Bazel target is:
+  // `//sw/device/tests:example_test_from_flash`, then you would specify this as
+  // `sw/device/tests/example_test_from_flash`.
   //
-  // Each entry in `sw_images` is followed by an index separated with ':' which
-  // is used by the testbench to know what type of image is it:
+  // Additionally, each entry in `sw_images` is followed by an index separated
+  // with ':' which is used by the testbench to know what type of image is it:
```

Comment:
Is it necessary to reformat the bazel labels?

Reply:
This was due to custom dvsim.py / DV testbench logic that parses these special file "labels". Specifically, this is a special custom file label format developed by the DV team that the testbench parses to understand where the SW image is located (in the build-out dir) and what type of SW image it is (i.e. ROM vs Flash) so it knows which memory to backdoor load it into. I can file an issue to better align this format with  bazel labels themselves. I just didn't want to make to many change to the DV testbench setup in this PR, instead wanted to focus on just swapping out the underlying build system to prevent hidden cross-cutting issues from materializing.